solidity/contracts/converter/BancorConverter.sol

```diff
@@ -394,12 +391,13 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
         validReserve(_reserveToken)
         returns (uint256, uint256)
     {
-        Reserve storage reserve = reserves[_reserveToken];
+        uint256 amount = IBancorFormula(addressOf(BANCOR_FORMULA)).calculatePurchaseReturn(

```

Comment:
@yudilevi : Perhaps a little, but that's not the reason for this change. In the `remove_ether_token` branch, due to one or more of our changes, we get a compilation stack-overflow error in some of these functions (`getPurchaseReturn` and/or `getSaleReturn` and/or `getCrossReserveReturn`). Since I wanted to apply as minimum changes as possible in the `remove_ether_token` branch, along with the fact that this change does not reflect the actual **functional** update that the `remove_ether_token` branch yields, I decide to "go back" and apply this change in here (the `0.5.19` branch).

Reply:
ok